<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VqbMainWindow</class>
 <widget class="QMainWindow" name="VqbMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>695</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>VqbForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelOutputVars">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>421</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output variables:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabQuery">
     <property name="statusTip">
      <string>Displays current query</string>
     </property>
     <attribute name="title">
      <string>Resulting Query</string>
     </attribute>
     <widget class="QTextEdit" name="queryViewer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>180</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabResults">
     <property name="statusTip">
      <string>Test-runs the query</string>
     </property>
     <attribute name="title">
      <string>Preview Query Results</string>
     </attribute>
     <widget class="QTextEdit" name="queryResults">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>180</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>740</width>
      <height>380</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>736</width>
       <height>376</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="outputList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonUp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>491</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move Up</string>
    </property>
    <property name="statusTip">
     <string>Moves variable up one position</string>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonDown">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move Down</string>
    </property>
    <property name="statusTip">
     <string>Moves variable down one position</string>
    </property>
   </widget>
   <widget class="KPushButton" name="buttonRemove">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>550</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove</string>
    </property>
    <property name="statusTip">
     <string>Removes variable</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="action_Return_to_Startup"/>
   </widget>
   <widget class="QMenu" name="menuWeb">
    <property name="title">
     <string>&amp;Web</string>
    </property>
    <addaction name="actionPostToPastebin"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuWeb"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPostToPastebin">
   <property name="text">
    <string>Post to pastebin.com</string>
   </property>
  </action>
  <action name="action_Return_to_Startup">
   <property name="text">
    <string>&amp;Return to Startup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>outputList</tabstop>
  <tabstop>buttonUp</tabstop>
  <tabstop>buttonDown</tabstop>
  <tabstop>buttonRemove</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>queryViewer</tabstop>
  <tabstop>queryResults</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
